sbnana/CAFAna/Systs/EnergySysts.h

```diff
@@ -0,0 +1,115 @@
+    EnergyScaleSystDetector detector;
+    double uncertainty;
+  };
+
+  extern const EnergyScaleSyst kEnergyScaleMuon;
+  extern const EnergyScaleSyst kEnergyScaleMuonSqrt;
+  extern const EnergyScaleSyst kEnergyScaleMuonInvSqrt;
+
+  extern const EnergyScaleSyst kEnergyScaleMuonND;
+  extern const EnergyScaleSyst kEnergyScaleMuonSqrtND;
+  extern const EnergyScaleSyst kEnergyScaleMuonInvSqrtND;
+
+  extern const EnergyScaleSyst kEnergyScaleMuonUB;
+  extern const EnergyScaleSyst kEnergyScaleMuonSqrtUB;
+  extern const EnergyScaleSyst kEnergyScaleMuonInvSqrtUB;
+
+  extern const EnergyScaleSyst kEnergyScaleMuonFD;
+  extern const EnergyScaleSyst kEnergyScaleMuonSqrtFD;
+  extern const EnergyScaleSyst kEnergyScaleMuonInvSqrtFD;
+
+  extern const EnergyScaleSyst kEnergyScaleHadron;
+  extern const EnergyScaleSyst kEnergyScaleHadronSqrt;
+  extern const EnergyScaleSyst kEnergyScaleHadronInvSqrt;
+
+  extern const EnergyScaleSyst kEnergyScaleHadronND;
+  extern const EnergyScaleSyst kEnergyScaleHadronSqrtND;
+  extern const EnergyScaleSyst kEnergyScaleHadronInvSqrtND;
+
+  extern const EnergyScaleSyst kEnergyScaleHadronUB;
+  extern const EnergyScaleSyst kEnergyScaleHadronSqrtUB;
+  extern const EnergyScaleSyst kEnergyScaleHadronInvSqrtUB;
+
+  extern const EnergyScaleSyst kEnergyScaleHadronFD;
+  extern const EnergyScaleSyst kEnergyScaleHadronSqrtFD;
+  extern const EnergyScaleSyst kEnergyScaleHadronInvSqrtFD;
+
+  extern const EnergyScaleSyst kEnergyScaleMuonBig;
+  extern const EnergyScaleSyst kEnergyScaleMuonSqrtBig;
+  extern const EnergyScaleSyst kEnergyScaleMuonInvSqrtBig;
+
+  extern const EnergyScaleSyst kEnergyScaleMuonNDBig;
+  extern const EnergyScaleSyst kEnergyScaleMuonSqrtNDBig;
+  extern const EnergyScaleSyst kEnergyScaleMuonInvSqrtNDBig;
+
+  extern const EnergyScaleSyst kEnergyScaleMuonUBBig;
+  extern const EnergyScaleSyst kEnergyScaleMuonSqrtUBBig;
+  extern const EnergyScaleSyst kEnergyScaleMuonInvSqrtUBBig;
+
+  extern const EnergyScaleSyst kEnergyScaleMuonFDBig;
+  extern const EnergyScaleSyst kEnergyScaleMuonSqrtFDBig;
+  extern const EnergyScaleSyst kEnergyScaleMuonInvSqrtFDBig;
+
+  extern const EnergyScaleSyst kEnergyScaleHadronBig;
+  extern const EnergyScaleSyst kEnergyScaleHadronSqrtBig;
+  extern const EnergyScaleSyst kEnergyScaleHadronInvSqrtBig;
+
+  extern const EnergyScaleSyst kEnergyScaleHadronNDBig;
+  extern const EnergyScaleSyst kEnergyScaleHadronSqrtNDBig;
+  extern const EnergyScaleSyst kEnergyScaleHadronInvSqrtNDBig;
+
+  extern const EnergyScaleSyst kEnergyScaleHadronUBBig;
+  extern const EnergyScaleSyst kEnergyScaleHadronSqrtUBBig;
+  extern const EnergyScaleSyst kEnergyScaleHadronInvSqrtUBBig;
+
+  extern const EnergyScaleSyst kEnergyScaleHadronFDBig;
+  extern const EnergyScaleSyst kEnergyScaleHadronSqrtFDBig;
+  extern const EnergyScaleSyst kEnergyScaleHadronInvSqrtFDBig;
+
+std::vector<const EnergyScaleSyst*> GetEnergySysts();
+std::vector<const EnergyScaleSyst*> GetBigEnergySysts();
```

Comment:
Are these better as `vector<const ISyst*>`? Would that make it easier to combine with other lists of systs, or does it not matter because `a.insert(a.end(), b.begin(), b.end())` works anyway if the types are compatible?